addons/website/models/mixins.py

```diff
@@ -285,7 +285,7 @@ def _search_website_published(self, operator, value):
     def open_website_url(self):
         return {
             'type': 'ir.actions.act_url',
-            'url': url_join(self.website_id.domain, self.website_url) if self.website_id else self.website_url,
+            'url': url_join(self.website_id.domain or '', self.website_url) if self.website_id else self.website_url,
```

Comment:
Wouldn't make it more easy to read? (no website record would still go through the else)
```suggestion
            'url': url_join(self.website_id.domain, self.website_url) if self.website_id.domain else self.website_url,
```

Reply:
I didn't know you could do that with the ORM! Thanks for the tip!